projects/valtimo/user-management/src/lib/user-list/user-list.component.html

```diff
@@ -16,33 +16,43 @@
 
 <div class="main-content pt-0">
   <div class="container-fluid">
-    <div class="text-right">
-      <div class="btn-group mt-m3px mb-3">
-        <button [routerLink]="'create'" class="btn btn-primary btn-space mr-0">
-          <i class="icon mdi mdi-plus mr-1"></i>
-          <span>{{ 'Create new User' | translate }}</span>
-        </button>
+    <div class="row mb-5" *ngIf="userManagementDisabled">
```

Comment:
Please run prettier formatting on all the code and don't forget to check for linting error

Reply:
done